File: c++/Disk.cpp
class Disk{
    private:
        //atribut
        string type;
        string capacity;
        int price;

    public:
        //konstruktor
        Disk(){
        }
        
        Disk(string type, string capacity, int price){
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        //setter
        void setType(string type){
            this->type = type;
        }

        void setCapacity(string capacity){
            this->capacity = capacity;
        }

        void setPrice(int price){
            this->price = price;
        }

        //getter
        string getType(){
            return this->type;
        }

        string getCapacity(){
            return this->capacity;
        }

        int getPrice(){
            return this->price;
        }

        void printDisk(){
            cout << "Disk~ " << endl;
            cout << "Type      : "<< this->type << endl;
            cout << "Capacity  : "<< this->capacity << endl;
            cout << "Price     : Rp "<< this->price << endl;
        }

        //destruktor
        ~Disk(){
        }

};

File: php/Disk.php
<?php 
 
 class Disk{

    //atribut
	private $type;
    private $capacity;
	private $price;
		
	public function __construct(){
	}
		
	//setter getter 
	public function setType($type){
		$this->type = $type;
	}
		
	public function getType(){
		return $this->type;
	}

    public function setCapacity($capacity){
		$this->capacity = $capacity;
	}

    public function getCapacity(){
		return $this->capacity;
	}
		
	public function setPrice($price){
		$this->price = $price;
	}
		
	public function getPrice(){
		return $this->price;
	}
	public function	printDisk(){
        echo "Disk~ <br>";
        echo "Type      : ".$this->type. "<br>";
        echo "Capacity  : ".$this->capacity."<br>";
        echo "Price     : Rp ".$this->price. "<br>";
    }
	function __destruct(){
	}
}
?>

		
		